cli/CMakeLists.txt

```diff
@@ -44,6 +44,20 @@ configure_file (
   "${CMAKE_CURRENT_BINARY_DIR}/CliConfig.h"
 )
 
+# Compile the protos for Single Node Server mode.
+if(ENABLE_SINGLE_NODE_SERVER)
+  # We will need some of the TMBs libraries
+  set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/third_party/src/tmb/cmake")
+
+  find_package(Grpc++ REQUIRED)
+  include_directories(${GRPCPLUSPLUS_INCLUDE_DIRS})
```

Comment:
Also good catch on keeping the orig module path

Note that cmake likes brackets:
```
set(CMAKE_MODULE_PATH
      ${CMAKE_MODULE_PATH}
      "${PROJECT_SOURCE_DIR}/third_party/src/tmb/cmake")
```

works while

```
set(CMAKE_MODULE_PATH
      $CMAKE_MODULE_PATH
      "${PROJECT_SOURCE_DIR}/third_party/src/tmb/cmake")
```
treats the middle argument like a string literal